test-support/trie-bench/Cargo.toml

```diff
@@ -13,6 +13,6 @@ trie-standardmap = { path = "../trie-standardmap", version = "0.15.2" }
 hash-db = { path = "../../hash-db" , version = "0.15.2"}
 memory-db = { path = "../../memory-db", version = "0.18.0" }
 trie-root = { path = "../../trie-root", version = "0.15.2" }
-trie-db = { path = "../../trie-db", version = "0.19.0" }
+trie-db = { path = "../../trie-db", version = "0.19.2" }
```

Comment:
oh, I see know, so the 0.18.0 version actually depends on `trie-db` `0.18.0`, not `0.19.0`
I think this change warrants a breaking release of `trie-bench`